src/main/java/appeng/api/hotkeys/LocatingService.java

```diff
@@ -0,0 +1,14 @@
+package appeng.api.hotkeys;
+
+import net.minecraft.world.entity.player.Player;
+
+/**
+ * locates and opens Inventories from items
+ */
+public interface LocatingService {
```

Comment:
Maybe we could go for a more general name? As far as I can tell, this can be used for any keybinding that can have multiple handlers. Maybe `HotkeyAction`?